### ISSUE TITLE
Controller vibration not working in HTML5

### ISSUE THREAD

**Issue by michaelgundlach** [godotengine/godot#96985]
### Tested versions

4.3 and earlier

### System information

Windows / OSX, Chrome

### Issue description

Input.start_joy_vibration() does nothing in HTML exports.

May be related to #73446 .

### Steps to reproduce

Export a game to HTML5 that runs Input.start_joy_vibration() and upload it to a webserver.  No vibration.

### Minimal reproduction project (MRP)

N/A

**Comment by Febrandt** [comment]
>Input.start_joy_vibration() does nothing in HTML exports.

Currently godot doesn't support the gamepad controller vibration for web, only Input.vibrate_handheld()

**Comment by michaelgundlach** [comment]
> Currently godot doesn't support the gamepad controller vibration for web, only Input.vibrate_handheld()

Right, this is what I am describing in this bug.  Either we add the support, or we at least document that it is missing.
